src/RadicleId.svelte

```diff
@@ -2,12 +2,12 @@
   import { parseRadicleId, toClipboard, twemoji } from "@app/lib/utils";
```

Comment:
We have a duplicate `RadicleId.svelte` now. The one in `/src/components/` should be the only one.

Reply:
Good catch, the rebasing is error-prone, hence why I suggest we all work on the same branch.

Reply:
[fixed](https://github.com/radicle-dev/radicle-interface/compare/946598481552437d95779585affe3a72b91c5012..0cb354754222e9aeda21703232cf7d112f528cd0)